website/docs/intro.md

```diff
@@ -76,7 +76,7 @@ Currently available plugins include:
 
 ### Taqueria SDK
 
-Taqueria has an SDK and protocol at it's core wich are dependencies for the CLI and all plugins. They are provided in the binary file, as well as plugins (npm packages)
+Taqueria has an SDK and protocol at its core which. All Taqueria plugins, and the cli depend on the Taqueria SDK
```

Comment:
Taqueria has an SDK and protocol at its core which all Taqueria plugins and the CLI depend on